### ISSUE TITLE
Newsletter schedule every X weeks or months

### ISSUE THREAD

**Issue by vesterli** [Simplero/Roadmap#254]
The current Newsletter feature only offers weekly and monthly cycles. I want to publish a newsletter every two weeks, so I suggest expanding the feature to allow cycles of "every X weeks" and every X months".

**Comment by themanagementcoach** [comment]
I support this request! I also send my newsletter out every 2 weeks, and would like to be able to choose that option.

**Comment by calvincorreli** [comment]
So what would the semantics be if you choose multiple weekdays, eg. Monday and Thursday, and then also say "every 2 weeks".

Monday + Thursday week 1, and then again Monday + Thursday week 3?

Or you can only pick one weekday if you want to not send out every week?

**Comment by themanagementcoach** [comment]
I wonder if the simplest fix for now is to add a button that says "Bi-weekly" in between "Weekly" and "Monthly".
I would think if you said "Every x weeks/days/months" you would probably not be able to select days of the week. I'm not sure how common it is amongst your customers to send out more than one newsletter in each week.

**Comment by calvincorreli** [comment]
You'd still want to select one weekday where you'd want it to be sent. Like changing it from check boxes to radio buttons.

**Comment by themanagementcoach** [comment]
Yes, I guess so. How does google calendar handle it? I think I’m able to select every two weeks and the days of the week. 

Mary Legakis Engel  |  647-985-9358 
The Management Coach 


On Feb 27, 2018, at 6:01 PM, Calvin Correli <notifications@github.com> wrote:

You'd still want to select one weekday where you'd want it to be sent. Like changing it from check boxes to radio buttons.
